eng/scripts/CodeCheck.ps1

```diff
@@ -89,7 +89,7 @@ try {
     Get-ChildItem "$repoRoot/*.sln" -Recurse `
         | ? {
             # These .sln files are used by the templating engine.
-            ($_.Name -ne "RazorComponentsWeb-CSharp.sln") -and ($_.Name -ne "GrpcService-CSharp.sln")
```

Comment:
Why is this removed?

Reply:
This was only required for multi-project templates that include a sln. This is now removed.